src/util/run-as-root.js

```diff
@@ -0,0 +1,42 @@
+const debug = require('./debug').here(__filename);
+const path = require('path');
+const { writeFile } = require('./promisified/fs');
+const sudoPrompt = require('sudo-prompt');
+const { join } = require('path');
+const tmp = () =>
+    join(
+        __dirname,
+        'tmp' +
+            Math.random()
+                .toString(20)
+                .slice(2)
+    );
+
+const opts = {
+    name: 'Magento PWA Studio',
+    icns: path.join(__dirname, '../../buildpack-logo.icns')
+};
+
+const elevate = async cmd =>
```

Comment:
`elevate` what? Name doesn't describe the purpose without reading through the implementation